src/app/shared/navigation-focus/navigation-focus.spec.ts

```diff
@@ -0,0 +1,103 @@
+import {Component, NgZone} from '@angular/core';
+import {ComponentFixture, fakeAsync, inject, TestBed, tick} from '@angular/core/testing';
+import {Router} from '@angular/router';
+import {RouterTestingModule} from '@angular/router/testing';
+import {NavigationFocusService} from './navigation-focus.service';
+
+describe('Navigation focus service', () => {
+  let navigationFocusService: NavigationFocusService;
+  let router: Router;
+  let zone: NgZone;
+  let fixture: ComponentFixture<NavigationFocusTest>;
+
+  const navigate = (url: string) => {
+    zone.run(() => router.navigateByUrl(url));
+    tick(100);
+  };
+
+  beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [RouterTestingModule.withRoutes([
+          {path: '', component: RouteTest},
+          {path: 'cdk', component: RouteTest},
+          {path: 'guides', component: RouteTest}
+        ])],
+        providers: [NavigationFocusService],
+        declarations: [NavigationFocusTest, RouteTest],
+      });
+      fixture = TestBed.createComponent(NavigationFocusTest);
+    }
+  );
+
+  beforeEach(inject(
```

Comment:
Can you try updating this to use `TestBed.inject` instead of `beforeEach(inject(...))`?
i.e.
```
beforeEach(() => {
  zone = TestBed.inject(NgZone);
etc
```

Reply:
Is this the new recommended way to do it? I've always done the `beforeEach(inject(...))` style

Reply:
I would say so, yea. If I recall correctly `beforeEach(inject(...))` was the way to do it before `TestBed.inject` existed. The testing guide https://angular.io/guide/testing uses `TestBed.inject` and the angular cli does as well when you generate a new service.